### ISSUE TITLE
FCM v1 Upgrade - 'NoneType' object has no attribute 'items'

### ISSUE THREAD

**Issue by bluepen50** [matrix-org/sygnal#368]
Running the latest version of Synapse (1.104.0) and Sygnal (0.14.0) and getting the following error:

```
 File "/usr/local/lib/python3.10/site-packages/sygnal/gcmpushkin.py", line 673, in _build_data
    for attr, value in data["content"].items():
AttributeError: 'NoneType' object has no attribute 'items'
```

This is from trying to upgrade to use the new V1 Firebase tokens- have updated the config so that is contains the following:
```
api_version: v1
project_id : id
service_account_file : filepath.json
```

What might be causing this issue?

**Comment by devonh** [comment]
This was fixed in #362
I can make a new release with the fix.

**Comment by devonh** [comment]
release 0.14.1 fixes this issue
